### PR TITLE
Reformat all source code according to the formatting rules. #424

### PR DESCRIPTION
**Pull request by rosrossito**

### CODE REVIEW

File: .IdeaIC14/config/codestyles/JavaParser-idea.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<code_scheme name="JavaParser">
```
Comment (matozoid) [inline .IdeaIC14/config/codestyles/JavaParser-idea.xml]: Can you move this file to /dev-files/ ?

**Comment by matozoid** [comment]
So if we format files with Eclipse and Intellij with these configuration files, do they differ a lot?

**Comment by ptitjes** [comment]
Having that in `dev-files/` would indeed permit to have a diff with the previous version...

**Comment by ptitjes** [comment]
And if it is not clear that those files are in the repo as explained at the bottom of the wiki page, maybe it should be quickly mentioned at its top.

I always have been cold about having ide-specific hidden directory in projects but that dates from when maven support in eclipse was not very good. So I may be wrong...

**Comment by rosrossito** [comment]
I moved file to folder /dev-files/.
Although on the wiki page it is mentioned that it must be in
.IdeaC14/config/codestyles...

**Comment by matozoid** [comment]
So, how are the results? Is formatting in intellij and eclipse identical now?

**Comment by matozoid** [comment]
This is not going anywhere.

**Pull request closed without merging**